frontend/javascripts/oxalis/view/layouting/flex_layout_wrapper.js

```diff
@@ -250,6 +250,11 @@ class FlexLayoutWrapper extends React.PureComponent<Props, State> {
       <FlexLayout.Layout
         model={model}
         factory={(...args) => this.layoutFactory(...args)}
+        titleFactory={renderedNode => (
+          <Tooltip title={BorderTabs[renderedNode.getId()].description}>
```

Comment:
For abbreviated tab titles, the tooltip should ideally show "Long Name - Description". For example, hovering over "AbsTree" should show a tooltip with "Abstract Tree - Shows ...".